tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_acme.yml

```diff
@@ -0,0 +1,36 @@
+- name: Installing ACME Responder
+  shell: pki-server acme-create --instance {{topology}}-CA
+  notify:
+    - INC_CONSTANTS
+
+- meta: flush_handlers
+
+- name: Configuring ACME Responder Database
+  copy:
+    src: /usr/share/pki/acme/conf/database/in-memory/database.conf
+    dest: /etc/pki/{{topology}}-CA/acme/database.conf
+    remote_src: yes
+    force: yes
+
+- name: Configuring ACME Responder Issuer
+  copy:
+    src: /usr/share/pki/acme/conf/issuer/pki/issuer.conf
```

Comment:
This was moved to `/usr/share/pki/acme/issuer/pki/issuer.conf`.

Reply:
Updated